File: src/org/bigraph/bigmc/model/BigraphPair.java
package org.bigraph.bigmc.model;

import org.bigraph.bigsim.model.Bigraph;


public class BigraphPair {
	protected Bigraph sourceBigraph;
    
    protected Bigraph targetBigraph;  
    
    public BigraphPair(Bigraph sourceBigraph, Bigraph targetBigraph) {
    	this.sourceBigraph = sourceBigraph;
    	this.targetBigraph = targetBigraph;
    }

	public Bigraph getSourceBigraph() {
		return sourceBigraph;
	}

	public void setSourceBigraph(Bigraph sourceBigraph) {
		this.sourceBigraph = sourceBigraph;
	}

	public Bigraph getTargetBigraph() {
		return targetBigraph;
	}

	public void setTargetBigraph(Bigraph targetBigraph) {
		this.targetBigraph = targetBigraph;
	}
    
}

File: src/org/bigraph/bigmc/model/BigraphPair.scala
package org.bigraph.bigmc.model

import org.bigraph.bigsim.model.Bigraph;
/**
 * @author amy
 */
object BigraphPair {
  
}

class BigraphPair {
   var sourceBigraph: Bigraph = null;
   var targetBigraph: Bigraph = null;
  
  def this(surceBigraph: Bigraph, targetBigraph: Bigraph) = {
    this();
    init(surceBigraph, targetBigraph);
  }
  def init(sourceBigraph: Bigraph, targetBigraph: Bigraph): Unit = {
    this.sourceBigraph = sourceBigraph;
    this.targetBigraph = targetBigraph;
  }
  
  override def toString = "sourceBigraph：" + sourceBigraph + "--->" + "\n" + "targetBigraph：" + targetBigraph + ";";
  
}
